client/src/components/CommentBody/CommentBody.css

```diff
@@ -1,19 +1,66 @@
+body {
+  text-align: center;
+}
```

Comment:
We don't want to have a global selector like this because it may clash with styles in other components